### PR TITLE
Essentia Condenser Nuke #1

### PR DESCRIPTION
**Pull request by Alastors**
As promised, the second something that could feasibly replace the condenser gets introduced, I'd nuke the condenser, here that is.

I'm going to leave the condenser itself in-tact as a block in-case someone actually wants to fix it in the future (this just removes the recipe, coremod PR inbound)

Sister PR to:
https://github.com/GTNewHorizons/MagicBees/pull/34
and
https://github.com/GTNewHorizons/NewHorizonsCoreMod/pull/779

Fixes:
https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/10522

### CODE REVIEW

File: src/main/resources/assets/gadomancy/lang/en_US.lang [hunk 1]
```diff
@@ -100,6 +100,7 @@ gadomancy.research_page.ESSENTIA_COMPRESSOR.4=By assembling a reinforced shell a
...
 gadomancy.research_page.ESSENTIA_COMPRESSOR.9=If an element of darkness is placed on an ancient pedestal near the condenser, the core of the condenser will consume it, allowing the condenser to increase its internal capacity by a reasonable amount. The highest point the capacity of the condenser can be pushed to with this method is 3000 per aspect.<LINE>You noticed while standing near the structure, that the pull of the Element of Darkness in the Condensers core is enough to gradually pull you closer. Could it be getting bigger? You don’t think so...hopefully...
+gadomancy.research_page.ESSENTIA_COMPRESSOR.4=No Recipe, sorry. Try the Apiamancer's Drainer.
```
Comment (OneEyeMaker) [inline src/main/resources/assets/gadomancy/lang/en_US.lang:103]:
```suggestion
gadomancy.research_page.ESSENTIA_COMPRESSOR.5=No Recipe, sorry. Try the Apiamancer's Drainer.
```

**Comment by EnderProyects** [comment]
If you remove the essentia condenser, you should change the creative essentia cell ( maybe use the new 16384k cells they where added)

**Comment by Dream-Master** [comment]
@Alastors is the new magic bees block a replacement of this one ?

**Comment by EnderProyects** [comment]
Also @Alastors could just be removed the formation of the multiblock ( is in the first lines of the recipes code if I remember correctly) , so that the blocks can still be crafted and used in the creative essentia recipe

**Comment by Alastors** [comment]
> @Alastors is the new magic bees block a replacement of this one ?

Not directly, it’s a replacement in spirit, but the condenser was a glitch that held up another glitch, this is a balanced bandaid. It’s not perfect, but frankly it’s enough to justify nuking the original glitch.

**Comment by Dream-Master** [comment]
@Alastors i set it draft

**Comment by Dream-Master** [comment]
not needed anymore

**Pull request closed without merging**